impeller/renderer/backend/metal/allocator_mtl.mm

```diff
@@ -194,6 +195,11 @@ static MTLStorageMode ToMTLStorageMode(StorageMode mode,
 
   mtl_texture_desc.storageMode = ToMTLStorageMode(
       desc.storage_mode, supports_memoryless_targets_, supports_uma_);
+  if (@available(macOS 12.5, ios 15.0, *)) {
+    if (mtl_texture_desc.storageMode == MTLStorageModePrivate) {
+      mtl_texture_desc.compressionType = MTLTextureCompressionTypeLossy;
```

Comment:
I'd like to attempt this in another patch and have this flag be present on `impeller::TextureDescriptor`.

Reply:
Makes sense, removed for now.